### PR TITLE
chore: remove benchmark devDependencies

### PR DESCRIPTION
**Pull request by RafaelGSS**
As pointed by @Eomm here: https://github.com/fastify/fastify/pull/3139#discussion_r660478998

We can also decide to use `npx` or remove `npx` in favor of `devDependencies` in that case.

#### Checklist

- [X] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [X] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by RafaelGSS** [comment]
I've increased the duration of the benchmark to get more _reliable_ results running.

**Review by mcollina — APPROVED** [review]
lgtm

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
 
       - name: Install
         run: |
-          npm install --ignore-scripts
+          npm install ---only=production
```
Comment (Eomm) [inline .github/workflows/benchmark.yml:49]:
```suggestion
          npm install --only=production --ignore-scripts
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ jobs:
 
       - name: Install
         run: |
-          npm install --ignore-scripts
+          npm install --only=production
```
Comment (Eomm) [inline .github/workflows/benchmark.yml:33]:
```suggestion
          npm install --only=production --ignore-scripts
```

**Review by Eomm — COMMENTED** [review]
I would keep the --ignore-scripts since we are checking out an external repo (the secrets should not be set anyway)

**Review by Eomm — APPROVED** [review]

**Pull request merged**